packages/angular/migrations.json

```diff
@@ -1469,6 +1469,46 @@
         "jest-preset-angular": {
           "version": "~12.2.0",
           "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/architect": {
```

Comment:
We need to add a migration generator to bump `@angular/cli`.

Reply:
A migration generator? So we can’t do it in this file anymore?